mailgun/tests.py

```diff
@@ -269,20 +269,156 @@ def test_empty_body_html(self, mock_ml_get, mock_ci_get, mock_ss_filter):
     @patch('mailgun.route_handlers.SuperSender.objects.filter')
     @patch('mailgun.route_handlers.CourseInstance.objects.get_primary_course_by_canvas_course_id')
     @patch('mailgun.route_handlers.MailingList.objects.get_or_create_or_delete_mailing_list_by_address')
-    def test_multi_mailing_list_recipients(self, mock_ml_get, mock_ci_get, mock_ss_filter):
+    def test_multi_mailing_list_recipients_with_course_settings_none(self, mock_ml_get, mock_ci_get, mock_ss_filter):
```

Comment:
Maybe we can update the docstring to describe the importance of the `_with_course_settings_none`?

We may not need all this anyway. `test_multi_mailing_list_recipients_with_always_mail_staff_true()` checks the behavior when `always_mail_staff==True`, so perhaps this could either reuse that test or simply check that `always_mail_staff` is indeed set to `True` when you send mail to a list that doesn't have a `CourseSettings` object...? Just want to avoid lots of duplication, and minimize the number of tests that check what the field defaults mean in the real world.